{"name": "test_someting2", "status": "failed", "statusDetails": {"message": "AssertionError: The sataus code is wrong", "trace": "self = <tests.test.crud.test_crud.TestCrud object at 0x000001679485C760>\nget_booking_id = 3844\n\n    def test_someting2(self,get_booking_id): ##Time pass ignore it\n        url=APIConstants.url_get_search(booking_id=get_booking_id)\n        print(url)\n        print(get_booking_id)\n        response=get_request(url=url,auth=None,in_json=False)\n        print(response.json())\n        actual_status_code=response.status_code\n>       verify_http_status_code(response_data=actual_status_code,expected_data=404)\n\ntest_crud.py:84: \n_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _\n\nresponse_data = 200, expected_data = 404\n\n    def verify_http_status_code(response_data, expected_data):\n>       assert response_data == expected_data ,\"The sataus code is wrong\"\nE       AssertionError: The sataus code is wrong\n\n..\\..\\..\\src\\helpers\\common_verification.py:2: AssertionError"}, "attachments": [{"name": "stdout", "source": "032c089b-397b-41d0-a84f-cba64796f6b4-attachment.txt", "type": "text/plain"}], "start": 1730195903580, "stop": 1730195904896, "uuid": "84c9c698-bfd7-4974-b2e5-c485a74ee476", "historyId": "f96621bee909390f9e3b4e6550927874", "testCaseId": "78c38a059a447b210768886c0f3862a4", "fullName": "tests.test.crud.test_crud.TestCrud#test_someting2", "labels": [{"name": "parentSuite", "value": "tests.test.crud"}, {"name": "suite", "value": "test_crud"}, {"name": "subSuite", "value": "TestCrud"}, {"name": "host", "value": "LAPTOP-J82A4UMN"}, {"name": "thread", "value": "7980-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.test.crud.test_crud"}]}